tlsobs-api/handlers.go

```diff
@@ -69,17 +80,74 @@ func ResultHandler(w http.ResponseWriter, r *http.Request) {
 		}
 	}()
 
-	domain := r.FormValue("id")
+	log := logger.GetLogger()
 
-	if validateDomain(domain) {
+	status = http.StatusInternalServerError
 
-		status = http.StatusOK
+	log.WithFields(logrus.Fields{
+		"form values": r.Form,
+		"headers":     r.Header,
+	}).Debug("Received request")
 
-	} else {
-		status = http.StatusBadRequest
+	val, ok := context.GetOk(r, dbKey)
+	if !ok {
+		log.Error("Could not find db in request context")
+		err = errors.New("Could not access database.")
+		return
+	}
+
+	db := val.(*pg.DB)
+
+	scanIDStr := r.FormValue("scan_id")
```

Comment:
same comment as for the certificate endpoint: we're already in the `scan` api namespace, so just calling it `id` is sufficient, since we already now we're requesting a scan